cyipopt/scipy_interface.py

```diff
@@ -385,7 +385,10 @@ def minimize_ipopt(fun,
                    constraints=(),
                    tol=None,
                    callback=None,
-                   options=None):
+                   options=None,
+                   mult_g=None,
+                   mult_x_L=None,
+                   mult_x_U=None,):
```

Comment:
This function is supposed to mirror the scipy `minimize()` function. The new arguments you have added are not available in SciPy's `minimize()` function. Shouldn't then these new arguments be passed in through the `options` keyword which allows you to provide options for a specific solver?

Reply:
I see. But I thought all keys in `options` are or could be transferred into options supported by ipopt (am I wrong?), wouldn't it be missleading if these three arguments are passed through `options`?